tests/test_branch_gen_answers.py

```diff
@@ -7,4 +7,5 @@
     "../taxonomy",
     "main",
 )
-mt_bench_branch.gen_answers("http://localhost:8000/v1")
+mt_bench_branch.gen_answers(server_url="http://localhost:8000/v1")
+mt_bench_branch.gen_answers(server_url="http://localhost:8000/v1", api_key="NO_API_KEY")
```

Comment:
Was there a need to change these "tests"?  Wouldn't they just accept the default if api_key isn't passed?

Reply:
yes, true - I'll remove these changes, they aren't needed

Reply:
updated & removed tests changes

Reply:
and squashed commits with more descriptive commit message